Hw2Task1/TrieTests/TrieTests.cs

```diff
@@ -0,0 +1,87 @@
+using NUnit.Framework;
+using Hw2Task1;
+
+namespace TrieTests;
+
+/// <summary>
+/// Module tests for trie 
+/// </summary>
+public class Tests
+{
+    private Trie testTrie = new Trie();
+    [SetUp]
+    public void Setup()
+    {
+        testTrie = new Trie();
+    }
+
+    [Test]
+    public void TrieShouldNotContainItemsAfterAddAndRemove()
+    {
+        testTrie.AddItem("test");
+        testTrie.AddItem("testtest");
+        testTrie.AddItem("test123");
+        testTrie.Remove("test");
+        Assert.IsTrue(testTrie.Contains("testtest") && testTrie.Contains("test123") && !testTrie.Contains("test"));
```

Comment:
Надо разбить на "элементарные" Assert-ы, чтобы сразу было видно, что пошло не так:
```suggestion
        Assert.IsTrue(testTrie.Contains("testtest"));
        Assert.IsTrue(testTrie.Contains("test123"));
        Assert.IsTrue(!testTrie.Contains("test"));
```